### PR TITLE
add remove old kubeedge task

### PR DESCRIPTION
**Pull request by pixiake**
## What this PR does / why we need it:
The old version kubeedge, if it exists, should be deleted, otherwise there will be a conflict between the old and new versions.

Signed-off-by: pixiake <guofeng@yunify.com>

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@wenchajun Please double check that `helm del kubeedge -n kubeedge` remove both the old kubeedge and the edgewatcher we created.

Maybe use kubesphere v3.2.1 to test

File: roles/edgeruntime/tasks/kubeedge.yaml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- name: KubeEdge | Remove old kubeedge
+  shell: >
+    {{ bin_dir }}/helm del kubeedge -n kubeedge
```
Comment (wenchajun) [inline roles/edgeruntime/tasks/kubeedge.yaml:3]: This I tested in my local environment, I used the command `helm del kubeedge -n kubeedge` and it showed that iptable was not uninstalled, when I used another command `helm uninstall kubeedge -n kubeedge`, it showed success, is it possible to use the command `uninstall`?

Comment (benjaminhuo) [reply roles/edgeruntime/tasks/kubeedge.yaml:3]: Both the EdgeWatcher deployment and the EdgeWatcherAgent daemonset are deleted after `helm uninstall kubeedge -n kubeedge` ?

Comment (pixiake) [reply roles/edgeruntime/tasks/kubeedge.yaml]: I have updated the tasks of removing old kubeedge.

**Review by wenchajun — APPROVED** [review]

File: roles/edgeruntime/tasks/kubeedge.yaml [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+    {{ bin_dir }}/helm list -f kubeedge -n kubeedge
+  register: kubeedge_check
+  failed_when: false
+
+- block:
+    - name: KubeEdge | Remove old kubeedge
+      shell: |
+        {{ bin_dir }}/helm uninstall kubeedge -n kubeedge
+        {{ bin_dir }}/kubectl patch iptables -n kubeedge iptables  -p '{"metadata":{"finalizers":null}}' --type merge
+        {{ bin_dir }}/kubectl delete ds -n kubeedge iptables
```
Comment (benjaminhuo) [inline roles/edgeruntime/tasks/kubeedge.yaml:13]: Maybe we need to remove CR of `iptablesrules` as well?

Comment (pixiake) [reply roles/edgeruntime/tasks/kubeedge.yaml:13]: I not found CR of `iptablesrules` in cluster. When will the CR of  `iptablesrules` be created?

Comment (benjaminhuo) [reply roles/edgeruntime/tasks/kubeedge.yaml:13]: When edge node is added

Comment (pixiake) [reply roles/edgeruntime/tasks/kubeedge.yaml:13]: I just tested it, and  CR of `iptablesrules` will be automatically deleted, which will not affect the uninstallation of kubeedge.

Comment (benjaminhuo) [reply roles/edgeruntime/tasks/kubeedge.yaml:13]: perfect!

**Pull request merged**